src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card className="text-left">
+      <CardContent>
+        <div class="row">
+          <h3>{title}</h3>
+        </div>
+        <div className="d-flex col-12">
+          <div class="row">
+            <div id="dg_main-content">
+              <img
+                className="dg_image left"
+                src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+                alt="County Executive Olzsewski delivers a speech."
+              />
+              <div className="d-flex dg_card_date-author">
+                <div className="dg_card_date-author_separator">
+                  <p>{published}</p>
+                </div>
+                <div>
+                  <p>{author}</p>
+                </div>
+              </div>
+              <p>{articleSummary}</p>
+            </div>
+          </div>
+        </div>
+      </CardContent>
+      <CardFooter>
+        <div class="row justify-content-end">
+          <div class="col-2">
+            <Button text="Details" />
```

Comment:
to make this a link this should be.

```jsx
<Button as="link" text="Details" href={url} />
```

We probably should added an example when we updated. Also going to add a owrk item for this because to me this seems confusing.

Reply:
as we discussed this throws an error when changing to a link.

![image](https://user-images.githubusercontent.com/40430541/74468808-759d5800-4e69-11ea-9fca-c75bd6528949.png)